### PR TITLE
[Document Intelligence] Fix CI pipeline

### PR DESCRIPTION
**Pull request by samvaity**

### CODE REVIEW

File: sdk/documentintelligence/azure-ai-documentintelligence/pom.xml [hunk 1]
```diff
@@ -15,35 +14,33 @@
   <groupId>com.azure</groupId>
   <artifactId>azure-ai-documentintelligence</artifactId>
   <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-ai-documentintelligence;current} -->
-  <packaging>jar</packaging>
 
-  <name>Microsoft Azure SDK for DocumentIntelligence</name>
-  <description>This package contains Microsoft Azure DocumentIntelligence client library.</description>
+  <name>Microsoft Azure client library for Document Intelligence</name>
+  <description>This package contains the Microsoft Azure Cognitive Services Document Intelligence SDK.</description>
```
[the remark this answers is not in the document]
Comment (samvaity) [reply sdk/documentintelligence/azure-ai-documentintelligence/pom.xml:19]: It is still being confirmed.

File: sdk/documentintelligence/azure-ai-documentintelligence/src/samples/README.md [hunk 2]
```diff
@@ -4,8 +4,7 @@ languages:
   - java
 products:
   - azure
-  - azure-cognitive-services
```
Comment (samvaity) [inline sdk/documentintelligence/azure-ai-documentintelligence/src/samples/README.md:7]: https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main

https://dev.azure.com/azure-sdk/public/_build/results?buildId=3258134&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=ac8f4042-9b76-5db4-27b1-2a4abaa9bb3c&l=18
?

Comment (mssfang) [reply sdk/documentintelligence/azure-ai-documentintelligence/src/samples/README.md:7]: https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/Test-SampleMetadata.ps1#L198

You will need to add the new slug in the link above.

Comment (mssfang) [reply sdk/documentintelligence/azure-ai-documentintelligence/src/samples/README.md:7]: https://github.com/Azure/azure-sdk-tools/blob/main/doc/common/common_engsys.md#workflow

**Review by HarshaNalluru — APPROVED** [review]

**Pull request merged**